README.md

```diff
@@ -26,9 +26,9 @@ sudo apt install libsqlite3-dev sqlite3 just cargo
 just build-release
 sudo just install
 ```
-_For Pop-os, write `COSMIC_DATA_CONTROL_ENABLED=1` in `/etc/environment`. (see [this issue](https://github.com/wiiznokes/clipboard-manager/issues/61))_
+Write `COSMIC_DATA_CONTROL_ENABLED=1` in `/etc/environment`. (see [this issue](https://github.com/wiiznokes/clipboard-manager/issues/61))_
```

Comment:
I think the italic will be broken without the first `_` ? Also, this workarround is not necessary for distro like fedora.

Reply:
Oh I see, also the workaround was necessary for me on fedora

Reply:
Which version ?

Reply:
kde 40 lastest

Reply:
Oops, didn't see the response, thanks for letting me know. I hope to find a better solution soon